pkg/coordinator/coordinator.go

```diff
@@ -66,28 +73,25 @@ func newReportCoordinator(r types.Registry, s time.Duration, cacheClean time.Dur
 	return c
 }
 
-// Filter returns a filter function that removes upkeep keys that apply to this
-// filter. Returns false if a key should be filtered out.
-func (rc *reportCoordinator) Filter() func(types.UpkeepKey) bool {
-	return func(key types.UpkeepKey) bool {
-		blockKey, id, err := key.BlockKeyAndUpkeepID()
+// IsPending returns false if a key should be filtered out.
```

Comment:
umm, should the name/behaviour be opposite?
If a key is pending then it should be filtered out?